app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/controllers.js");
+
+app.use(express.json());
```

Comment:
This should not be used until you have POST/PATCH methods being handled. Remove it for now 👍